src/scripts/migrate.py

```diff
@@ -9,8 +9,7 @@
 - https://incatools.github.io/ontology-access-kit/
 - https://incatools.github.io/ontology-access-kit/intro/tutorial02.html
 
-TODO's (major):
-  -
+todo: refactor to take in 'unmapped mapppable' terms i.e. reports/%_unmapped_terms.tsv instead of other params?
```

Comment:
FYI: Idea: It may be possible to take in `reports/%_unmapped_terms.tsv ` (generated in step 1 of the mapping progress monitor) as a parameter to `slurp` instead of several other params, simplifying it.

Reply:
Good idea! Make sure this dependency is explicit in the makefile if you decide to implement this in a future PR